src/elements/SelectInput.vue

```diff
@@ -14,10 +14,10 @@ withDefaults(defineProps<Props>(), {
   disabled: false,
 });
 
-defineEmits(['submit']);
+defineEmits(["submit"]);
```

Comment:
I thought we had a rule to use single quotes? Guess it's not eslint, or was never formalized 😅